numbat/src/value.rs

```diff
@@ -26,7 +28,7 @@ pub enum Value {
     Boolean(bool),
     String(String),
     /// A DateTime with an associated offset used when pretty printing
-    DateTime(chrono::DateTime<chrono::Utc>, chrono::FixedOffset),
+    DateTime(chrono::DateTime<chrono::FixedOffset>),
```

Comment:
@eminence Do you remember why you chose to store datetime in UTC + offset separately?

Reply:
No, not really.   I think the initial version didn't support any datetime conversions, so storing a `DateTime<Utc>` was a natural fit, and it didn't evolve in the best way when Tz conversions were added.  I may have also been operating under [a potentially faulty] assumption that it would be easier to reason about datetime operations is everything was done in Utc (and then only converted to something else for printing). 

However skimming your new version here, it does look less complicated 👍

Reply:
Thank you!